module/VuFind/src/VuFind/Search/Factory/ProQuestFSGBackendFactory.php

```diff
@@ -113,7 +113,7 @@ protected function createBackend(Connector $connector)
      */
     protected function createConnector()
     {
-        $connector = new Connector($this->createHttpClient());
+        $connector = new Connector($this->createHttpClient(), $this->proQuestFSGConfig->toArray() ?? []);
```

Comment:
toArray is guaranteed to return an array, so no need for null coalescing.

```suggestion
        $connector = new Connector($this->createHttpClient(), $this->proQuestFSGConfig->toArray());
```